Resources/Prototypes/_NF/Entities/Objects/Consumable/Food/produce.yml

```
@@ -410,3 +410,35 @@
   - type: DeepFrySpawn
     cycles: 2
     output: FoodOnionRings
+
+- type: entity
+  name: monkana
+  parent: FoodProduceBase
+  id: FoodMonkana
+  description: Looks like a monkeyfied banana.
+  components:
+  - type: FlavorProfile
+    flavors:
+      - monkana
+  - type: SolutionContainerManager
+    solutions:
+      food:
+        maxVol: 25
+        reagents:
+        - ReagentId: JuiceThatMakesYouOOOK
```

```suggestion
        - ReagentId: JuiceThatMakesYouOok
```
If accepting the OOOK to Ook change, here's the other use.